CHANGELOG.md

```diff
@@ -23,6 +23,12 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#1061](https://github.com/realm/SwiftLint/issues/1061)
 
+* Add 'ignores_function_declarations' and 'ignores_comments' as options
+  to LineLengthRule.
```

Comment:
missing two trailing spaces after `.`. Also, please scape `ignores_function_declarations` in backticks (\`)